apps/frontend/src/libs/components/avatar/avatar.tsx

```diff
@@ -0,0 +1,41 @@
+import { type JSX } from "react";
+
+import { Image } from "../components.js";
+import styles from "./styles.module.css";
+
+type Properties = {
+	user: User;
+};
+
+type User = {
+	avatarUrl: null | string;
+	firstName: string;
+	lastName: string;
+};
+
+const getInitials = (user: User): string => {
+	const [firstChar] = user.firstName;
+	const [lastChar] = user.lastName;
+
+	const firstNameInitial = firstChar ? firstChar.toUpperCase() : "";
+	const lastNameInitial = lastChar ? lastChar.toUpperCase() : "";
+
+	return `${firstNameInitial}${lastNameInitial}`;
+};
+
+const Avatar = ({ user }: Properties): JSX.Element => {
+	return (
+		<div className={styles["avatar"]}>
+			<Image
+				alt="User Avatar"
+				className={styles["avatarImage"] ?? ""}
```

Comment:
let's use typecast to reconcile it with TypeScript. We will follow the next principle:

If this is the case, that can happen in code by its logic - we add js checks/safeguards. 

If by logic it's not possible or not expected, but typescript requires it - we will use typecast.

```suggestion
				className={styles["avatarImage"] as string}
```